Fix build failure when an old version of qpdf is installed and active.
https://github.com/qpdf/qpdf/issues/158
--- examples/build.mk.orig
+++ examples/build.mk
@@ -39,4 +39,4 @@ $(foreach B,$(CBINS_examples),$(eval \
 
 $(foreach B,$(BINS_examples) $(CBINS_examples),$(eval \
   examples/$(OUTPUT_DIR)/$(call binname,$(B)): $(OBJS_$(B)) ; \
-	$(call makebin,$(OBJS_$(B)),$$@,$(LDFLAGS) $(LDFLAGS_libqpdf),$(LIBS_libqpdf) $(LIBS))))
+	$(call makebin,$(OBJS_$(B)),$$@,$(LDFLAGS_libqpdf) $(LDFLAGS),$(LIBS_libqpdf) $(LIBS))))
--- libtests/build.mk.orig
+++ libtests/build.mk
@@ -43,4 +43,4 @@ $(foreach B,$(BINS_libtests),$(eval \
 
 $(foreach B,$(BINS_libtests),$(eval \
   libtests/$(OUTPUT_DIR)/$(call binname,$(B)): $(OBJS_$(B)) ; \
-	$(call makebin,$(OBJS_$(B)),$$@,$(LDFLAGS) $(LDFLAGS_libqpdf),$(LIBS) $(LIBS_libqpdf))))
+	$(call makebin,$(OBJS_$(B)),$$@,$(LDFLAGS_libqpdf) $(LDFLAGS),$(LIBS) $(LIBS_libqpdf))))
--- qpdf/build.mk.orig
+++ qpdf/build.mk
@@ -30,4 +30,4 @@ $(foreach B,$(CBINS_qpdf),$(eval \
 
 $(foreach B,$(BINS_qpdf) $(CBINS_qpdf),$(eval \
   qpdf/$(OUTPUT_DIR)/$(call binname,$(B)): $(OBJS_$(B)) ; \
-	$(call makebin,$(OBJS_$(B)),$$@,$(LDFLAGS) $(LDFLAGS_libqpdf),$(LIBS_libqpdf) $(LIBS))))
+	$(call makebin,$(OBJS_$(B)),$$@,$(LDFLAGS_libqpdf) $(LDFLAGS),$(LIBS_libqpdf) $(LIBS))))
